editor/tests/org.fusesource.ide.wsdl2rest.ui.tests.integration/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,24 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %Bundle-Name
+Bundle-SymbolicName: org.fusesource.ide.wsdl2rest.ui.tests.integration
+Bundle-Version: 10.3.0.qualifier
```

Comment:
as version has been upgraded will need to modify to 11.0.0 (and rebase on master)